GetIntoTeachingApi/Services/Store.cs

```diff
@@ -270,7 +270,7 @@ private async Task<Point> CoordinateForPostcode(string postcode)
 
         private async Task CacheLocation(string postcode, Point coordinate)
         {
-            var location = new Location(postcode, coordinate);
+            var location = new Location(postcode, coordinate, Source.Google);

```

Comment:
This needs a test to cover the source being correctly set to `Google` when we're caching a geocoded response; can probably add it in here:

https://github.com/DFE-Digital/get-into-teaching-api/blob/master/GetIntoTeachingApiTests/Services/StoreTests.cs#L128

You can just change the last line:

```
DbContext.Locations.FirstOrDefault(l => l.Postcode == sanitizedPostcode && l.Source == Source.Google).Should().NotBeNull();
```